content/resources/help-centre/software-tips/path.md

```diff
@@ -0,0 +1,96 @@
+---
+title: The PATH variable
+---
+
+## What is the "PATH"?
+
+If you have been following the lessons and help guides available on this
+website, and installing software such as FFmpeg and AP (AnalysisPrograms) then
+you have probably seen the term `PATH` by now. 
+
+The `PATH` variable is a system wide variable (known as en environment variable)
```

Comment:
Not strictly true. 

Windows has two path environment variables that get merged: a system one and a per-user one.

```suggestion
The `PATH` variable an environment variable. Environment variables are values that 
```